label_sync/labels.yaml

```diff
@@ -882,26 +882,107 @@ repos:
   kubernetes/k8s.io:
     labels:
       - color: 0052cc
-        description: Issues or PRs related to defining who has access to what (e.g. IAM, RBAC, google groups)
+        description: Define who has access to what (e.g. IAM, RBAC, google groups)
         name: area/access
         target: both
         prowPlugin: label
         addedBy: anyone
+        deleteAfter: 2021-08-04
+      - color: 0052cc
+        description: Application management, code in apps/
+        name: area/apps
+        target: both
+        prowPlugin: label
+        addedBy: anyone
+        previously:
+          - name: area/cluster-infra
+      - color: 0052cc
+        description: cert-manager, code in apps/cert-manager/
+        name: area/apps/cert-manager
```

Comment:
Do we want to have this until cert-manager is decommissioned ?

Reply:
I'm not sure if we want to remove this once cert manager is gone or not.  

Keeping it:
- pro: we keep historical data for e.g. "how much work did we save ourselves by moving to a hosted solution"
- con: we have additional label clutter

Dropping it:
- pro: we have fewer labels to choose from
- con: we lose historical data

I'm inclined to keep it for now.  We could also consider an `area/cert-management` or something to cover the area as a whole vs. one specific implementation

Reply:
Let's keep it for historical tracking.